torch/csrc/distributed/c10d/quantization_gpu.h

```diff
@@ -0,0 +1,9 @@
+// (c) Facebook, Inc. and its affiliates. Confidential and proprietary.
+
+#pragma once
+
+#include <ATen/ATen.h>
+#include <vector>
+
+at::Tensor _float_to_bfloat16_gpu(const at::Tensor& input);
```

Comment:
same, wrap this in some namespace, global namespace is not a good practice in C++